rollup.config.js

```diff
@@ -0,0 +1,115 @@
+import alias from 'rollup-plugin-alias'
+import babel from 'rollup-plugin-babel'
+import commonjs from 'rollup-plugin-commonjs'
+import resolve from 'rollup-plugin-node-resolve'
+import peerDepsExternal from 'rollup-plugin-peer-deps-external'
+import replace from 'rollup-plugin-replace'
+import { terser } from 'rollup-plugin-terser'
+
+const path = require('path')
+const fs = require('fs')
+const { join } = require('path')
+const { readdirSync } = require('fs-extra')
+
+const NODE_ENV = process.env.NODE_ENV || 'development'
+const extensions = ['.web.js', '.js']
+
+function removeExt(path) {
+  return path.replace(/\.[^.]+$/, '')
+}
+
+const res = readdirSync('src/components').reduce((acc, filename) => {
```

Comment:
What does `res` mean ? What does `acc` mean ? Maybe giving explicit names to the variables will help understand the code better imo :)

Reply:
Improved